jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -138,7 +137,7 @@ pipeline {
                         )}"""
                 K8S_VERSION= """${sh(
                         returnStdout: true,
-                        script: 'cd ${TESTDIR}; gke/get-cluster-version.sh'
+                        script: 'cat ${TESTDIR}/gke/cluster-version'
```

Comment:
I think that GKE build is failing because this env var value contains newline at the end. Please add `| tr '\n' ''` to this `cat` call

Reply:
Eh... sure, fixed now. I think this should be just an API call instead of env var that is set externally this way.

Reply:
https://github.com/cilium/cilium/issues/11189